opencti-platform/opencti-graphql/tests/01-unit/database/modelConsistency-test.js

```diff
@@ -466,3 +467,15 @@ describe('Testing stix ref extractor', () => {
     expect(refs.includes('marking-definition--34098fce-860f-48ae-8e50-ebd3cc5e41da')).toBe(true);
   });
 });
+
+describe('Test if needed', () => {
```

Comment:
it's not really a clear name for a describe

Reply:
oups ...totally forgot 😇